clients/privacy-center/config/config.json

```diff
@@ -9,11 +9,10 @@
       "policy_key": "default_access_policy",
       "icon_path": "/download.svg",
       "title": "Access your data",
-      "description": "We will provide you a report of all your personal data.",
+      "description": "Please provide an email. We will provide you a report of all your personal data.",
       "identity_inputs": {
         "name": "optional",
```

Comment:
Similar to phone, I'd like us to remove `name` here as well, since it won't be used for the execution, etc.